java-openjdk-8/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fedora:21
```

Comment:
might be better to have it as `fedora` (always points to the latest stable release) instead of `fedora:21`